src/integration-test/tests/legal/certificateOfService/steps/certificateOfService.ts

```diff
@@ -0,0 +1,20 @@
+import { WhatDocumentsPage } from '../pages/what-documents'
+import { DocumentUploadPage } from '../pages/document-upload'
+
+const whatDocumentsPage: WhatDocumentsPage = new WhatDocumentsPage()
+const documentUploadPage: DocumentUploadPage = new DocumentUploadPage()
+
+export class CertificateOfServiceSteps {
+  // todo not used, remove?
+  open () {
+    documentUploadPage.open()
+  }
+
+  // todo not used, remove?
+  uploadDocument () {
```

Comment:
I guess this is not used because feature is toggled off.

Reply:
done